featuremanagement/aio/_featuremanager.py

```diff
@@ -26,6 +27,8 @@ class FeatureManager:
 
     :param Mapping configuration: Configuration object.
     :keyword list[FeatureFilter] feature_filters: Custom filters to be used for evaluating feature flags.
+    :keyword Callable[EvaluationEvent] on_feature_evaluated: Callback function to be called when a feature flag is
```

Comment:
Is there any discussion about this? Currently, `on_feature_evaluated` will only take effect when featureflag.telemetry.enabled == true. This looks weird to me because the callback function is called "on feature evaluated", it's supposed to be called each time the feature flag is evaluated.

Reply:
The `on_feature_evaluated` was the named that was decided on during the design review meeting. I understand it's a bit weird but coming up with a name that fully describes what it does while keeping a reasonable keyword length is a challenge. I'm open up to hear options.

As is, this will go out in Beta for a while before GA, so we'll have time to change it.

Reply:
Resolving this we have a larger conversation bellow. https://github.com/microsoft/FeatureManagement-Python/pull/24#discussion_r1673464625